birdhouse/config/thredds/entrypointwrapper

```diff
@@ -20,8 +20,12 @@ fi
 # https://www.unidata.ucar.edu/software/tds/v4.6/reference/TomcatBehindProxyServer.html
 WEBAPPS_ROOT="/usr/local/tomcat/webapps"
 EXISTING_CONTEXT_ROOT="thredds"
-WANTED_CONTEXT_ROOT="twitcher/ows/proxy/thredds"
-WANTED_CONTEXT_ROOT_WARFILE_NAME="twitcher#ows#proxy#thredds"
+if [ -z "$WANTED_CONTEXT_ROOT" ]; then
+    WANTED_CONTEXT_ROOT="twitcher/ows/proxy/thredds"
+fi
+if [ -z "$WANTED_CONTEXT_ROOT_WARFILE_NAME" ]; then
+    WANTED_CONTEXT_ROOT_WARFILE_NAME="twitcher#ows#proxy#thredds"
+fi
```

Comment:
Not sure of both variable names starting with `WANTED_`. 
Would only `CONTEXT_ROOT` do the job instead?

Reply:
Existing variable name, I did not want to change the existing code too much.  There is `EXISTING_CONTEXT_ROOT` and this `WANTED_CONTEXT_ROOT`.  Does this naming really bother you a lot?

Reply:
Not at all. Seemed strange when I first saw it but I would probably not even notice if I haven't reviewed this part.